Data/symbol-properties/symbol-properties-database.csv

```diff
@@ -103,8 +103,34 @@ usa,6N,future,,USD,100000,0.0001,1
 usa,6R,future,,USD,2500000,0.000005,1
 usa,6L,future,,USD,100000,0.00005,1
 usa,6M,future,,USD,500000,0.00001,1
+usa,6Z,future,,USD,500000,0.000025,1
+# TODO: verify if ACD is accurate
```

Comment:
What is needed to verify this?

Reply:
I'd like a second opinion regarding that symbol's minimum-price-variation. The CME website lists two potential variations, and I didn't know which one to choose.

I put a link to the symbol's specs in the original commit message. Check it out and let me know what your thoughts on it are :+1:

Reply:
Seems I've put the wrong currency for that symbol and the minimum-price-variation is wrong as well. Just found a page with all the details I need. Fix will be coming out shortly